package.json

```diff
@@ -33,25 +33,28 @@
     "@semantic-release/npm": "7.1.3",
     "@semantic-release/release-notes-generator": "9.0.3",
     "@types/fs-extra": "9.0.11",
-    "@types/jest": "26.0.23",
-    "@types/jest-image-snapshot": "4.3.0",
-    "@types/puppeteer": "5.4.3",
+    "@types/jest": "29.5.11",
+    "@types/jest-image-snapshot": "6.4.0",
+    "@types/puppeteer": "7.0.4",
+    "@typescript-eslint/eslint-plugin": "^6.20.0",
+    "@typescript-eslint/parser": "^6.20.0",
     "chalk": "4.1.1",
+    "eslint": "^8.56.0",
     "husky": "6.0.0",
-    "jest": "26.6.3",
-    "jest-image-snapshot": "4.5.0",
-    "prettier": "2.3.0",
+    "jest": "29.7.0",
+    "jest-image-snapshot": "6.4.0",
+    "prettier": "3.2.4",
     "semantic-release": "17.4.4",
     "trace": "3.1.1",
-    "ts-jest": "26.5.6",
-    "ts-node": "9.1.1",
-    "typescript": "4.2.4"
+    "ts-jest": "29.1.2",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.3.3"
   },
   "dependencies": {
     "commander": "7.2.0",
     "fs-extra": "10.0.0",
     "joi": "17.4.0",
-    "puppeteer": "9.1.1",
-    "xterm": "4.12.0"
+    "xterm": "4.12.0",
```

Comment:
The only non-internal/tooling change here is for `puppeteer`. I wonder if we should do this in #10 instead? since it is a product change, is only used with/impacts `xterm`. Otherwise, I'm fine with keeping the existing version if it works well.

Reply:
will keep here unless necessary

Reply:
Sounds good. Thanks.